specification/ai/Face/models.session.tsp

```diff
@@ -0,0 +1,187 @@
+  @doc("Unique ID to reference this session.")
+  sessionId: string;
+
+  @doc("Authorization token for use by the client application")
+  authToken: string;
+}
+
+@doc("Session status.")
+enum SessionStatus {
+  @doc("Session has not started.")
+  NotStarted,
+
+  @doc("Session has started.")
+  Started,
+
+  @doc("Session has available result.")
+  ResultAvailable,
+}
+
+@doc("Audit entry for a request in the session.")
+model SessionAuditEntryRequestInfo {
+  @doc("The URL of the request.")
+  url: string;
+
+  @doc("The HTTP method of the request.")
+  method: string;
+
+  @doc("The length of the request body in bytes.")
+  contentLength?: int64;
+
+  @doc("The content type of the request.")
+  contentType: string;
+
+  @doc("The user agent of the request.")
+  userAgent?: string;
+}
+
+@doc("Audit entry for a response in the session.")
+model SessionAuditEntryResponseInfo {
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Represent complex JSON object"
+  @doc("The response body.")
+  body: Record<unknown>;
+
+  @doc("The HTTP status code of the request.")
+  statusCode: int32;
+
+  @doc("The latency of the request.")
+  latencyInMilliseconds: int64;
+}
+
+@doc("Audit entry for a request in session.")
+@resource("audit")
+model BaseSessionAuditEntry {
+  @doc("ID of this audit entry.")
+  @key
+  @visibility("read")
+  id: int64;
+
+  @doc("Session ID of this audit entry.")
+  sessionId: string;
+
+  @doc("Request ID in the request header.")
+  requestId: string;
+
+  @doc("Client request ID in the request header.")
+  clientRequestId: string;
+
+  @doc("The UTC date time of the request.")
+  receivedDateTime: utcDateTime;
+
+  @doc("The request of this entry.")
+  request: SessionAuditEntryRequestInfo;
+
+  @doc("The response of this entry.")
+  response: SessionAuditEntryResponseInfo;
+
+  @doc("The digest of the request body.")
+  digest: string;
+}
+
+@parentResource(LivenessSession)
+model LivenessSessionAuditEntry is BaseSessionAuditEntry;
+
+@parentResource(LivenessWithVerifySession)
+model LivenessWithVerifySessionAuditEntry is BaseSessionAuditEntry;
+
+#suppress "@azure-tools/typespec-autorest/unsupported-multipart-type" "Provide convienient interface for multipart/form-data in SDK"
+@doc("Dedicated parameter model for multipart/form-data.")
+model LivenessSessionCreationContentForMultipart {
+  ...LivenessSessionCreationContent;
+}
+
+@doc("Request of liveness with verify session creation.")
+model LivenessSessionWithVerifyImageCreationContent {
+  @doc("The content type for the operation. Always multipart/form-data for this operation.")
+  @header("content-type")
+  contentType: "multipart/form-data";
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style"
+  @doc("The CreationContent for creating session.")
+  CreationContent: LivenessSessionCreationContentForMultipart;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style"
+  @doc("The image stream for verify.")
+  VerifyImage: bytes;
```

Comment:
Why did you switch to PascalCase for these two properties?

Reply:
That's reflecting the underlying key inside the form-data request of our existing service. Do you prefer to name these properties with camelCase and use decorator to specify it's PascalCase in actual request?